pkg/vm/builtins/keccak.go

```diff
@@ -22,6 +22,8 @@ const cellsPerKeccak = 16
 const inputCellsPerKeccak = 8
 
 type Keccak struct {
+	ratio                       uint64
+	instancesPerComponentKeccak uint64
```

Comment:
why is instancesPerComponent a membre of the builtin type struct for keccak only? it is not constant in this case?

Reply:
you never set a value to it so it will be 0?

Reply:
Yes, thats right! All of those values were meant to be moved to consts, since they do not change. Thats a good catch!

Reply:
Moved from the struct